docs/contents.md

```diff
@@ -23,8 +23,8 @@
 
 | Playbook Name                                  | Description                                                                                                             |
 |:-----------------------------------------------|:------------------------------------------------------------------------------------------------------------------------|
-| Check for SLA violation<sup>New</sup>          | Checks for breach notification violation and update SLA status to Missed                                                |
-| Reminder for SLA Breach<sup>New</sup>          | Sends reminder for the notification SLA Breach                                                                          |
+| Check for SLA violation                        | Checks for breach notification violation and update SLA status to Missed                                                |
+| Reminder for SLA Breach                        | Sends reminder for the notification SLA Breach                                                                          |
```

Comment:
This playbook is repeated in the table.  Refer to line no 37

Reply:
Fixed @cs-rohit-khune